utils/testdata/fuzz/FuzzDurationJSONRoundtrip/771e938e4458e983

```diff
@@ -0,0 +1,2 @@
+go test fuzz v1
```

Comment:
should we delete this file then?

Reply:
so this scenario will continue to be tested, so I think it's fine to leave in as is. see https://github.com/viamrobotics/rdk/pull/4511/files#r1825152416 for more details